src/s1/t6/n1/exercici1/Main.java

```diff
@@ -0,0 +1,21 @@
+package s1.t6.n1.exercici1;
+
+public class Main {
+        public static void main(String[] args) {
+                // Demonstrate that arguments can be passed in any order to the constructor
+                NoGenericMethods container1 = new NoGenericMethods("Hello", 123, true);
+                NoGenericMethods container2 = new NoGenericMethods(123, true, "Hello");
+                NoGenericMethods container3 = new NoGenericMethods(true, "Hello", 123);
+
+                // Print the objects from each container to verify order doesn't matter
```

Comment:
Could add a .toString() to your NoGericMethods class and save some typing/repetition.

Reply:
Yeah that's true but i only learned about `@override` in t7 👀 .